components/collector/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Row, SimpleSpotlightNFT } from './types'
+import formatBalance from '@/utils/formatBalance'
+import * as store from '~/store'
+import { getVolume, pairListBuyEvent } from '@/utils/math'
+
+export const nftFn = (a: any): Row => {
+  const sold = a.nfts.nodes.reduce(soldFn, 0)
+  const unique = a.nfts.nodes.reduce(uniqueFn, new Set()).size
+  const total = a.nfts.totalCount
+  const uniqueCollectors = a.nfts.nodes.reduce(
+    uniqueCollectorFn,
+    new Set()
+  ).size
+  const buyEvents = a.nfts.nodes.map(onlyEvents).map(pairListBuyEvent).flat()
+  const volume = Number(getVolume(buyEvents))
+
+  return {
+    id: a.issuer,
+    total,
+    sold,
+    unique,
+    averagePrice:
+      a.nfts.nodes.filter(onlyOwned).reduce(sumFn, 0) /
+      (a.nfts.nodes.length || 1),
+    count: 1,
+    collectors: 0, // a.nfts.nodes.reduce(uniqueCollectorFn, new Set()),
```

Comment:
i assume there's no property in the response to calculate this right? can we do a TODO comment here then to solve this in the future